client/consensus/manual-seal/src/lib.rs

```diff
@@ -19,7 +19,13 @@
 //! A manual sealing engine: the engine listens for rpc calls to seal blocks and create forks.
 //! This is suitable for a testing environment.
 
+#![allow(dead_code)]
+
+use std::{sync::Arc, marker::{PhantomData, Send}};
```

Comment:
```suggestion
use std::{sync::Arc, marker::PhantomData};
```

`Send` is in prelude.